server/jamovi/common/column.h

```diff
@@ -37,8 +37,10 @@ typedef struct
 {
     int value;
     int capacity;
+    int importCapacity;
     int count;
     char *label;
+    char *importValue;
```

Comment:
I didn't understand the capacity thing. I saw you calculate it but i couldn't find anywhere it being used.